src/doc/book/getting-started.md

```diff
@@ -166,8 +166,13 @@ you can find the Rust executables in a directory like
 `"C:\Program Files\Rust stable GNU 1.x\bin"`.
 
 Rust does not do its own linking, and so you’ll need to have a linker
-installed. Doing so will depend on your specific system, consult its
-documentation for more details.
+installed. Doing so will depend on your specific system. For
+Linux-based systems, Rust will attempt to call `cc` for linking. On
+`windows-msvc` (Rust built on windows with Microsoft Visual Studio),
+this depends on having Microsoft Visual Studio installed with its
+tools in `%PATH%`, namely `linker.exe`. However, if you have your
```

Comment:
`link.exe`. Also it doesn't need to be in `PATH` because rustc can find msvc itself.